src/core/evaluator.js

```diff
@@ -3376,13 +3419,21 @@ class PartialEvaluator {
         properties = {
           type,
           name: baseFontName,
+          loadedName: baseDict.loadedName,
           widths: metrics.widths,
           defaultWidth: metrics.defaultWidth,
           flags,
           firstChar,
           lastChar,
         };
         const widths = dict.get("Widths");
+        const standardFontName = getStandardFontName(baseFontName);
+        let file = null;
+        if (standardFontName) {
+          const data = await this.fetchStandardFontData(standardFontName);
+          file = new Stream(data);
```

Comment:
Please let `fetchStandardFontData` return the data as a `Stream`, rather than doing that manually at the call-sites.